packet/templates/active_packets.html

```diff
@@ -1,7 +1,5 @@
 {% extends "extend/base.html" %}
 
-{% set can_sign = (info.onfloor and info.uid != packet.rit_username) or info.realm == "csh" %}
```

Comment:
Should replace this with `info.onfloor or info.realm == "csh"` as used below, then restore the `can_sign` usage on line 23 and remove the new definition on line 28.